Game/Entities/Projectile.cs

```diff
@@ -14,6 +22,9 @@ public override void _PhysicsProcess(double delta)
 		{
 			QueueFree();
 		}
+
+		// Set the rotation of the sprite to match the direction of the projectile
+		sprite.Rotation = velocity.Angle();
```

Comment:
This is fine for now but ideally I think we would be rotating the projectile itself (and changing our projectile movement logic to fit)

Reply:
Fixing this requires using a rotation pivot, we can discuss if we want to add that!

Reply:
Probably just leave for now, we can sort it out later when we do more projectile stuff